conf/application.conf

```diff
@@ -38,23 +40,30 @@ play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-application.secret="jIk4O9ZvQkKdIgAuSYctYsdxPCTfz0wpeJ6kTBZmdcdxaHTug4gq9p4Yv2VqncPX"
+application.secret = "jIk4O9ZvQkKdIgAuSYctYsdxPCTfz0wpeJ6kTBZmdcdxaHTug4gq9p4Yv2VqncPX"
 
 # Session configuration
 # ~~~~~
-application.session.httpOnly=false
+application.session.httpOnly = false
 
-application.session.secure=false
+application.session.secure = false
 
 # The application languages
 # ~~~~~
-application.langs="en"
+application.langs = "en"
+
+
+play.modules.enabled += "uk.gov.hmrc.servicedependencies.SchedulerModule"
+
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule",
```

Comment:
I don't think you need auth and audit. 
There must be a way to configure a microservice to work without this modules. If not we need to do some changes. The HttpClientModule requires audit, so that can't be used either.

Reply:
Yes, auth is now removed from service-dependencies but audit (as you pointed out) can't be removed because JsonErrorHandler (and may be others) needing it:
`class JsonErrorHandler @Inject()(val configuration: Configuration, auditConnector: AuditConnector)`